geniza/corpus/models.py

```diff
@@ -457,6 +457,9 @@ class Document(ModelIndexable, DocumentDateMixin):
     fragments = models.ManyToManyField(
         Fragment, through="TextBlock", related_name="documents"
     )
+    image_order_override = ArrayField(
+        models.URLField(), null=True, verbose_name="Image Order"
```

Comment:
Let's add some help text here. Maybe something like this?
> Drag image thumbnails to customize order when necessary (i.e. image sequence does not follow fragment/shelfmark sequence)

Reply:
Ended up needing to override the `ModelAdmin.get_form` method to get this to show up in the form, since I use a read-only field based on the `admin_images` property for the actual reordering, while the `image_order_override` field is a hidden input.